pkg/proxy/iptables/proxier.go

```diff
@@ -378,13 +378,27 @@ var iptablesJumpChains = []iptablesJumpChain{
 	{utiliptables.TableNAT, kubePostroutingChain, utiliptables.ChainPostrouting, "kubernetes postrouting rules", nil},
 }
 
-var iptablesCleanupOnlyChains = []iptablesJumpChain{}
+func getExistingChains(ipt utiliptables.Interface, table utiliptables.Table) (chains []utiliptables.Chain, err error) {
+	iptablesData := bytes.NewBuffer(nil)
+	if err = ipt.SaveInto(table, iptablesData); err != nil {
+		klog.Errorf("Failed to execute iptables-save for %s: %v", table, err)
+	} else {
```

Comment:
return early if err != nil and remove this else?

Reply:
ok